Styles/Materials.js

```diff
@@ -17,10 +17,39 @@
  */
 
 import lng from '@lightningjs/core';
-import { CORNER_RADIUS } from './Styles';
 
 export const spacing = multiplier => 8 * multiplier;
 
+export const ALPHA = {
+  overlays: {
+    background: 72,
+    bubble: 56,
+    shelf: 56
+  },
+  text: {
+    primary: 100,
+    secondary: 72,
+    tertiary: 56,
+    inactive: 40
+  }
+};
+
+export const CORNER_RADIUS = {
+  large: 24,
+  medium: 16,
+  small: 8,
+  xsmall: 4
+};
```

Comment:
Alpha and corner radius don't belong in Materials.js - I would take a look at the themeing work that Josh is working on and try to set up the Styles folder similar to that so it's easier to roll over to that work